client/extensions/woocommerce/store-sidebar/store-ground-control.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+const Gridicon = require( 'gridicons' ),
+	Site = require( 'blocks/site' );
+
+import Button from 'components/button';
+
+const StoreGroundControl = ( { site, translate } ) => {
+	const backLink = '/stats/day/' + site.slug;
+
+	return (
+		<div className="store-sidebar__ground-control">
+			<Button
+				borderless
+				className="store-sidebar__ground-control-back"
+				href={ backLink }
+				aria-label={ translate( 'Go back' ) }
+			>
+				<Gridicon icon="arrow-left" />
+			</Button>
+			<div className="store-sidebar__ground-control-site">
+				<Site
+					compact
+					site={ site }
+					indicator={ false }
+					homeLink={ true }
```

Comment:
@DanReyLop Recently taught us that the true lines can be simplified to just `homeLink` and `externalLink`.
In JSX, an attribute without value is assumed to be `={ true }`.